src/Mollie/WC/Helper/PaymentMethodsIconUrl.php

```diff
@@ -45,25 +40,6 @@ public function svgUrlForPaymentMethod($paymentMethodName)
         return $this->fallToAssets($paymentMethodName);
```

Comment:
I would suggest moving `fallToAssets()` method body to this method and remove `fallToAssets()` method at all.

This is because there is actually nothing to fall from, using local assets is the only option now, so that name is a bit confusing. And another reason is this method currently does nothing except unconditionally calls another method. This is not a big deal, though, just a small optimization.